mantle/kola/tests/rhcos/luks.go

```diff
@@ -1,36 +1,21 @@
 package rhcos
 
 import (
-	"bytes"
 	"encoding/base64"
 	"fmt"
-	"io"
-	"io/ioutil"
-	"os"
-	"os/exec"
-	"os/user"
 	"regexp"
-	"strconv"
-	"sync"
-	"syscall"
 	"time"
 
 	"github.com/coreos/mantle/kola/cluster"
 	"github.com/coreos/mantle/kola/register"
+	"github.com/coreos/mantle/platform"
 	"github.com/coreos/mantle/platform/conf"
+	"github.com/coreos/mantle/platform/machine/unprivqemu"
 	"github.com/coreos/mantle/util"
 )
 
-var runOnce sync.Once
-
-const (
-	dbDir          = "/tmp/tang/db"
-	cacheDir       = "/tmp/tang/cache"
-	xinetdConfFile = "/tmp/tang/xinetd.conf"
-	xinetdPidFile  = "/tmp/tang/pid"
-	tangLogFile    = "/tmp/tang/tang.log"
-	xinetdLogFile  = "/tmp/tang/xinetd.log"
-)
+var thumbprint []byte
+var tangAddress string
```

Comment:
Not a huge fan of having these as package level globals. Any reason to not just localize them inside the test scope?

Reply:
I'll localize them